examples/symfony.php

```diff
@@ -45,6 +46,13 @@ public function json()
 
         return print_r($payload, true);
     }
+
+    public function valuesArray()
+    {
+        $name = $this->queryParameter('arr')->commaSeparated()->int();
```

Comment:
> I am leaning towards the multiple classes for each array type, what do you think? That would be the best way to maintain the library as consistent as possible in terms of capabilities.

Hm, yeah, there's presumably no way around it. So let's try out the multiple-class approach :)